File: fapi/src/main/java/com/netcracker/cinema/fapi/model/Ticket.java
package com.netcracker.cinema.fapi.model;

public class Ticket {
    private Long id;
    private int rowNumber;
    private int placeNumber;
    private boolean isSold;
    private double cost;

    public Long getId() {
        return id;
    }

    public void setId(Long id) {
        this.id = id;
    }

    public int getRowNumber() {
        return rowNumber;
    }

    public void setRowNumber(int rowNumber) {
        this.rowNumber = rowNumber;
    }

    public int getPlaceNumber() {
        return placeNumber;
    }

    public void setPlaceNumber(int placeNumber) {
        this.placeNumber = placeNumber;
    }

    public boolean getIsSold() {
        return isSold;
    }

    public void setIsSold(boolean isSold) {
        this.isSold = isSold;
    }

    public double getCost() {
        return cost;
    }

    public void setCost(double cost) {
        this.cost = cost;
    }
}

File: frontend/src/app/models/Ticket.ts
import {User} from "./User";
import {Session} from "./Session";

export class Ticket {
  id: number;
  rowNumber: number;
  placeNumber: number;
  cost: number;
  session: Session;
  user: User;

  constructor(rowNumber: number, placeNumber: number, cost: number, session: Session, user: User) {
    this.placeNumber = placeNumber;
    this.rowNumber = rowNumber;
    this.cost = cost;
    this.session = session;
    this.user = user;
  }
}
